ScreenControl/Assets/ScreenControl/Client/Scripts/CoreConnection/WebSocketReceiverQueue.cs

```diff
@@ -19,19 +19,25 @@ void Update()
             ClientInputAction action;
             while (receiveQueue.Count > queueCullLimit)
             {
-                receiveQueue.TryDequeue(out action);
-
-                // Stop skipping through the queue this frame if we have just handled a 'key' input event
-                if (action.InputType != InputType.MOVE)
+                if (receiveQueue.TryPeek(out action))
```

Comment:
Do we need this peek? Isn't testing that there's something there covered by the content of the `while`?

Reply:
The content of the while won't have run if we are at the bottom of the method and the condition of the while was never met